### ISSUE TITLE
Expanding "Create a New Preset" crashes Grabber and the Desktop

### ISSUE THREAD

**Issue by queuemm** [Bionus/imgbrd-grabber#3081]
**Steps to reproduce**

1. Go to Sources
2. Click on Add a new preset
3. Click and drag on the side of the "Create a new preset" window
4. Crashes Grabber and KDE Plasma Desktop with all other programs running.

[main.log](https://github.com/Bionus/imgbrd-grabber/files/13896074/main.log)
[settings-ini.txt](https://github.com/Bionus/imgbrd-grabber/files/13896077/settings-ini.txt)

**Screenshots**

![Screenshot_20240110_171011](https://github.com/Bionus/imgbrd-grabber/assets/93687469/260823a4-fe89-4fcd-b0c0-28b2bfd3af45)

**System information**

- OS: Arch Linux using Zen Kernel 6.6.10, with KDE Plasma 5.27.10 on Wayland
- Grabber version: 7.11.2 (x86)

**Additional context**

I complied the program from the AUR using "imgbrd-grabber" as the package with no modifications.

This is also my first bug report for GitHub, so I apologize if I missed a step or two.

**Comment by Bionus** [comment]
It crashes _the whole desktop_? 😱 
I just tried and couldn't reproduce (Windows), strange.

The code for this pop-up in question is quite straightforward and makes use of only Qt's API:
https://github.com/Bionus/imgbrd-grabber/blob/91068776f3dca51dad0f2087692b01a191a1ddd8/src/gui/src/sources/sources-window.cpp#L401

Could it be something between KDE and Qt? (since it crashes the whole desktop) Maybe it's fixed on the nightly version which uses a more recent Qt version? 🤔

**Comment by queuemm** [comment]
Yeah I suspect it's a QT/KDE only issue. But I can replicate it easily every time.

**Comment by Bionus** [comment]
I guess I'd need to boot up some KDE VM to reproduce then, since I believe they use modified Qt libs on their systems 🤔 
Although hopefully it's already fixed in the [nightly version](https://github.com/Bionus/imgbrd-grabber/releases/nightly) which uses Qt 6.6 instead of 5.15.
